server/models/Item.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize, DataTypes } = require('sequelize');
+const sequelize = require('../db');
+
+const Item = sequelize.define('Item' , {
+     id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      description: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      price: {
+        type: DataTypes.FLOAT,
+        allowNull: false
+      },
+      qty: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+      },
+      imageUrl: {
+        type: DataTypes.STRING,
+        allowNull: true
+      },
+      categoryId: {
```

Comment:
remove categoryId field, as it is unnecessary to define categoryId if you already defined relationships in index file, please see the docs: 
https://sequelize.org/docs/v6/core-concepts/assocs/#one-to-many-relationships
Also, refer to Options (link down below), to set foreign key constraint for mandatory relationships
https://sequelize.org/docs/v6/core-concepts/assocs/#options-1